### PR TITLE
what's new: notification center icon and menu basic view

### PR DESCRIPTION
**Pull request by japie1235813**
* Motivation for features / changes
Add mat-icon and mat-menu for notification center

* Technical description of changes
The notification content is a placeholder

* Screenshots of UI changes
Screen Shot 2021-03-01 at 2 05 20 PM
Screen Shot 2021-03-01 at 2 05 27 PM

### CODE REVIEW

File: tensorboard/webapp/notification_center/_views/notification_center_component.ng.html [hunk 1]
```diff
@@ -14,3 +14,19 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
+<button
+  class="mat-focus-indicator mat-icon-button mat-button-base"
+  mat-icon-button
+>
+  <span class="mat-button-wrapper">
+    <mat-icon
+      class="mat-icon mat-icon-no-color notranslate"
```
Comment (stephanwlee) [inline tensorboard/webapp/notification_center/_views/notification_center_component.ng.html:23]: These class names feel very odd to me. If you look at the Angular doc <https://material.angular.io/components/button/examples>, you would not need those classes.

image

File: tensorboard/webapp/notification_center/_views/notification_center_component.ng.html [hunk 2]
```diff
@@ -14,3 +14,19 @@
...
+      svgIcon="notifications_none_24px"
+      [matMenuTriggerFor]="menu"
```
Comment (stephanwlee) [inline tensorboard/webapp/notification_center/_views/notification_center_component.ng.html:25]: A button, or an interactive element, is a trigger, not a visual SVG. Put this on the `button`.

File: tensorboard/webapp/notification_center/_views/notification_center_component.ng.html (hunk 1 above)
Comment (japie1235813) [reply tensorboard/webapp/notification_center/_views/notification_center_component.ng.html:23]: Classes of which element (<button> or <mat-icon>)?
I'm following other icons' classes:
Screen Shot 2021-03-01 at 3 55 43 PM



Without the classes in <button> the ui breaks
Screen Shot 2021-03-01 at 3 53 11 PM

Without the classes in mat-icon
Screen Shot 2021-03-01 at 3 55 08 PM

Comment (stephanwlee) [reply tensorboard/webapp/notification_center/_views/notification_center_component.ng.html:23]: Do you have a branch that I can play with? This is surprisingly; it should not happen this way.

File: tensorboard/webapp/notification_center/_views/notification_center_component.ng.html (hunk 2 above)
Comment (japie1235813) [reply tensorboard/webapp/notification_center/_views/notification_center_component.ng.html]: Thanks! Done.

File: tensorboard/webapp/notification_center/_views/notification_center_component.ng.html (hunk 1 above)
Comment (japie1235813) [reply tensorboard/webapp/notification_center/_views/notification_center_component.ng.html]: Thanks for the feedback and pr. Used MatButtonModule instead.

**Review by stephanwlee — APPROVED** [review]

**Pull request merged**